rbac/management/querysets.py

```diff
@@ -206,6 +227,7 @@ def get_access_queryset(request):
             APPLICATION_KEY: app,
             "prefetch_lookups_for_ids": "resourceDefinitions",
             "prefetch_lookups_for_groups": "policies__roles__access",
+            "is_org_admin": is_org_admin,
```

Comment:
I _think_ we should be able to simplify this and just use `request.user.admin` without the need for any of the other changes in this file, since we add the user (and the admin flag from the identity header) onto the request object in the middleware: 

https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/rbac/middleware.py#L176
https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/rbac/middleware.py#L215